tests/test_sanitize.py

```diff
@@ -33,6 +33,14 @@ def test_strip_html(self):
             sanitize.strip_html('<foo>bar</foo>'),
             'bar'
         )
+        assert_equal(
+            sanitize.strip_html(b'<foo>bar</foo>'),
+            'bar'
+        )
+        assert_true(sanitize.strip_html(True))
+        assert_false(sanitize.strip_html(False))
+        assert_equal(sanitize.strip_html({}), {})
+        assert_equal(sanitize.strip_html([]), [])
```

Comment:
In general I try to avoid putting too many kinds of tests in the same function- if one thing fails, other failure reasons downstream aren't checked.

At minimum, could you break this up into two tests? (one for things that are strings, and one for things that are not)?